### PR TITLE
016/solution

### PR DESCRIPTION
**Pull request by BenGitsCode**
DO NOT merge this into master.

This PR exists only for review purposes, please close PR and delete branch if it passes review.

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -1,78 +1,94 @@
...
 
 ```bash
-// your response here
+bundle exec rails server
 ```
 
-What are the commands to drop, create and migrate a database from the command
-line? (3 bullet points)
+What are the commands to drop, create and migrate, and seed a database from the command
+line? (5 bullet points)
```
Comment (jrhorn424) [inline diagnostic.md]: You've made the change to 5 bullet points here, but not on master. Please update `016/master`, #107

**Review by jrhorn424 — COMMENTED** [review]
Please address changes on **`016/master`**. Then, I will update solution and close this PR.

File: diagnostic.md (hunk 1 above)
Comment (BenGitsCode) [reply diagnostic.md]: Addressed in 31cf4243276acb2cb081f155c130fbaa5e8e956c

**Comment by jrhorn424** [comment]
See how this has unresolved conflicts? Please fix from `016/master` or, since this was updated, `master`.

- on master, `git pull master`
- `git checkout 016/solution`
- `git rebase master`
- `git push`
- `git push --force-with-lease`

Then ping me.

**Comment by BenGitsCode** [comment]
@jrhorn424 Updated.

File: diagnostic.md [hunk 2]
```diff
@@ -1,78 +1,94 @@
-# Rails API Diagnostic
+\# Rails API Diagnostic
```
Comment (jrhorn424) [inline diagnostic.md]: Please fix.

**Pull request closed without merging**